src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureAnalyzer.cs

```diff
@@ -102,11 +112,11 @@ public override void Initialize(AnalysisContext context)
                     );
 
                 // Handle library side definitions of preview features
-                context.RegisterSymbolAction(context => AnalyzeSymbol(context, requiresPreviewFeaturesSymbols, requiresPreviewFeaturesSymbolsToUsageType, runtimeFeatureTypeVirtualStaticsInInterfaces), s_symbols);
+                context.RegisterSymbolAction(context => AnalyzeSymbol(context, requiresPreviewFeaturesSymbols, requiresPreviewFeaturesSymbolsToUsageType, virtualStaticsInInterfaces, previewFeaturesAttribute), s_symbols);
             });
         }
 
-        private static bool ProcessTypeSymbolAttributes(ITypeSymbol symbol, ConcurrentDictionary<ISymbol, bool> requiresPreviewFeaturesSymbols, ConcurrentDictionary<ISymbol, PreviewFeatureUsageType> requiresPreviewFeaturesSymbolsToUsageType)
+        private static bool ProcessTypeSymbolAttributes(ITypeSymbol symbol, ConcurrentDictionary<ISymbol, bool> requiresPreviewFeaturesSymbols, ConcurrentDictionary<ISymbol, PreviewFeatureUsageType> requiresPreviewFeaturesSymbolsToUsageType, INamedTypeSymbol previewFeatureAttributeSymbol)
```

Comment:
NIT: preferably lines completely visible for review
```suggestion
        private static bool ProcessTypeSymbolAttributes(ITypeSymbol symbol, ConcurrentDictionary<ISymbol, bool> requiresPreviewFeaturesSymbols,
         ConcurrentDictionary<ISymbol, PreviewFeatureUsageType> requiresPreviewFeaturesSymbolsToUsageType, INamedTypeSymbol previewFeatureAttributeSymbol)
```